tee-worker/omni-executor/client-sdk/packages/client-sdk/src/lib/requests/__test__/pumpx-add-export-wallet.request.test.ts

```diff
@@ -0,0 +1,61 @@
+import { enclave } from '@lib/enclave';
+import { PumpxRpcMethod } from '@lib/utils/rpc-method';
+import { JsonRpcRequest } from '@utils/types';
+
+// This test is just an example. It requires receiving email verification codes, which cannot be done by running this unit test.
+describe.skip('add-export-wallet', () => {
+  // step 1
+  it('request email verification code', async () => {
+    const rpcRequest = {
+      jsonrpc: '2.0',
+      method: 'omni_requestEmailVerificationCode',
+      params: ['test@google.com'],
+    };
+    await enclave.send(rpcRequest);
+  });
+
+  // step 2
+  it('request email verification code', async () => {
+    const rpcRequest: JsonRpcRequest = {
+      jsonrpc: '2.0',
+      method: 'pumpx_requestJwt',
+      params: ['test@google.com', '', '', '', '1223'],
+    };
+
+    const response = await enclave.send(rpcRequest);
+    // console.log(response.auth_token);
+  });
+  it('add wallet should works', async () => {
```

Comment:
Does this example really need this call?

Reply:
The F/E team already knows how to call it, but I added it anyway, just for reference.

Reply:
I mean that in order to export wallet you don't need to add anything - it's there :)

Reply:
ok,writing example code in a separate file is clearer.